pkg/tcl/testworkflowstcl/testworkflowexecutor/executor.go

```diff
@@ -148,6 +151,22 @@ func (e *executor) Recover(ctx context.Context) {
 	}
 }
 
+func (e *executor) updateStatus(execution *testkube.TestWorkflowExecution) {
+	if execution.TestWorkflowExecutionName != "" {
```

Comment:
Maybe better to check only once if the execution exists in Kubernetes? It may be called unnecessarily too often

Reply:
good point